src/main/java/com/artipie/settings/YamlStorage.java

```diff
@@ -40,6 +41,8 @@ public Storage storage() {
         if (Strings.isNullOrEmpty(type)) {
             throw new IllegalArgumentException("Storage type cannot be null or empty.");
         }
-        return new Storages().newStorage(type, this.yaml);
+        return new JfrStorage(
```

Comment:
I do not think it's correct to use `JfrStorage` as we should not measure anything if it's not configured

Reply:
JFR recording can be switched on at any time, so we always have to create wrapper.